oeps/best-practices/oep-0011/decisions/0002-redux.rst

```diff
@@ -0,0 +1,37 @@
+Use Redux
+#########
+
+Status
+******
+
+??? people are debating this one
```

Comment:
I think marking this old decision as accepted is fine.  This also sets up the stage for an official discussion about dropping redux via a future ADR that supersedes this one.